docs/data_classes.rst

```diff
@@ -0,0 +1,427 @@
+  >>> my_sequence.meta
+  {'Description': 'This is some sample NDCubeSequence metadata.'}
+
+The `~ndcube.NDCube` instances are stored in ``my_sequence.data`` while the metadata is stored at ``my_sequence.meta``.
+If we wanted to define a common axis, we must set it during instantiation.
+Let's reinstantiate the `~ndcube.NDCubeSequence` with the common axis as the first cube axis.
+
+.. code-block:: python
+
+  >>> my_sequence = NDCubeSequence([cube0, cube1, cube2, cube3], common_axis=0)
+
+.. _dimensions:
+
+Dimensions and Physical Types
+-----------------------------
+
+Analagous to `ndcube.NDCube.dimensions`, there is also a `ndcube.NDCubeSequence.dimensions` property for easily inspecting the shape of an `~ndcube.NDCubeSequence` instance.
+
+.. code-block:: python
+
+  >>> my_sequence.dimensions
+  (<Quantity 4. pix>, <Quantity 4. pix>, <Quantity 4. pix>, <Quantity 5. pix>)
+
+Slightly differently to `ndcube.NDCube.dimensions`, `ndcube.NDCubeSequence.dimensions` returns a tuple of `astropy.units.Quantity` instances with pixel units, giving the length of each axis.
+To see the dimensionality of the sequence in the cube-like paradigm, i.e. taking into account the common axis, use the `ndcube.NDCubeSequence.cube_like_dimensions` property.
+
+.. code-block:: python
+
+  >>> my_sequence.cube_like_dimensions
+  <Quantity [16., 4., 5.] pix>
+
+Equivalent to `ndcube.NDCube.array_axis_physical_types`, `ndcube.NDCubeSequence.array_axis_physical_types` returns a list of tuples of physical axis types.
+The same `IVOA UCD1+ controlled words <http://www.ivoa.net/documents/REC/UCD/UCDlist-20070402.html>`_ are used for the cube axes.
+The sequence axis is given the label ``'meta.obs.sequence'`` as it is the IVOA UCD1+ controlled word that best describes it.
+To call, simply do:
+
+.. code-block:: python
+
+  >>> my_sequence.array_axis_physical_types
+  [('meta.obs.sequence',),
+   ('custom:pos.helioprojective.lat', 'custom:pos.helioprojective.lon'), ('custom:pos.helioprojective.lat', 'custom:pos.helioprojective.lon'),
+   ('em.wl',)]
+
+Once again, we can see the physical types associated with each axis in the cube-like paradigm be calling `ndcube.NDCubeSequence.cube_like_array_axis_physical_types`.
+
+.. code-block:: python
+
+  >>> my_sequence.cube_like_array_axis_physical_types
+  [('custom:pos.helioprojective.lat', 'custom:pos.helioprojective.lon'),
+   ('custom:pos.helioprojective.lat', 'custom:pos.helioprojective.lon'),
+   ('em.wl',)]
+
+Explode Along Axis
+------------------
+During analysis of some data --- say of a stack of images --- it may be necessary to make some different fine-pointing adjustments to each image that isn't accounted for the in the original WCS translations, e.g. due to satellite wobble.
+If these changes are not describable with a single WCS object, it may be desirable to break up the N-D sub-cubes of an `~ndcube.NDCubeSequence` into an sequence of sub-cubes with dimension N-1.
```

Comment:
```suggestion
If these changes are not describable with a single WCS object, it may be desirable to break up the N-D sub-cubes of an `~ndcube.NDCubeSequence` into a sequence of sub-cubes with dimension N-1.
```